microsoft-edge/webview2/release-notes.md

```diff
@@ -70,18 +70,21 @@ Async methods:
 
 
 <!-- ====================================================================== -->
-## 1.0.1722.32
+## 1.0.1722.45
 
-Release Date: April 10, 2023
+Release Date: April 13, 2023
 
-[NuGet package for WebView2 SDK 1.0.1722.32](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1722.32)
+[NuGet package for WebView2 SDK 1.0.1722.45](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1722.45)
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1722.32 or higher.
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1722.45 or higher.
 
 
 <!-- ------------------------------ -->
 #### General
 
+> [!IMPORTANT]
+> **Breaking Change**: WebView2 release package 1.0.1722.32 is deprecated, and is removed with the next release.  Discontinue development with package 1.0.1722.32.  If your WebView2 app uses the package, we recommend that you move to a newer package.

```

Comment:
@novac42 @oggy22 @peiche-jessica - 
Review each phrase:
* Is this a breaking change?
* Will this SDK be removed with [in] the next release?
* Should they discontinue using .32?
* Do we need lead-in "If your ... uses"?
* Should we specify "newer" package?

Reply:
- Is this a breaking change? _I don't know if I will call it a breaking change considering .32 is already broken._
- Will this SDK be removed with [in] the next release? _There isn't an actual way to really remove a package but to un-list it. I have unlisted .32 i.e. it will not show up in the search result but one can still access it if they know the link. Though we do want to prompt developers to move to the new .45._
- Should they discontinue using .32? _They should._
- Do we need lead-in "If your ... uses"? _We can - we can say we recommend you use .45 instead._
- Should we specify "newer" package? _See above._

Reply:
snapshot of revised wording: I'm changing from:
> **Breaking Change**: WebView2 release package 1.0.1722.32 is deprecated, and is removed with the next release. Discontinue development with package 1.0.1722.32. If your WebView2 app uses the package, we recommend that you move to a newer package.
to:
> WebView2 release package 1.0.1722.32 is deprecated, and will be removed from the listing at NuGet in the next release, though the NuGet link for that package will continue to work.  Discontinue development with package 1.0.1722.32.  If your WebView2 app uses the package, we recommend that you move to a newer package, such as 1.0.1722.45 or later.